### ISSUE TITLE
Restore jit.util.funck

### ISSUE THREAD

**Issue by ryanoutcome20** [Facepunch/garrysmod-requests#2420]
### Details
Restore [jit.util.funck](https://wiki.facepunch.com/gmod/jit.util.funck), as its functionality is nearly identical to [debug.getlocal](https://wiki.facepunch.com/gmod/debug.getlocal), with a few minor differences. And it is very helpful for custom debuggers and error detection, and it only has access to Lua defined functions (so it poses zero security risk).

There is no reason to remove perfectly safe, functional, and useful functions as a "preemptive measure", especially when the function has no access to write and only to read.

**Comment by Zaurzo** [comment]
Yes please, bring back my homie jit.util.funck back from the dead
